ui/src/i18n/en.json

```diff
@@ -25,7 +25,8 @@
                 "rating": "Rating",
                 "quality": "Quality",
                 "bpm": "BPM",
-                "playDate": "Last Played"
+                "playDate": "Last Played",
+                "createdAt": "Created at"
```

Comment:
Please translate it as "Date Added", to be consistent with the Smart Playlist field.